distribution/kernel/carbon-home/repository/resources/conf/infer.json

```diff
@@ -38,7 +38,8 @@
       "user_store.properties.LDAPConnectionTimeout": 5000,
       "user_store.properties.ReplaceEscapeCharactersAtUserLogin": true,
       "user_store.properties.SCIMEnabled": false,
-      "user_store.properties.UserRolesCacheEnabled": true
+      "user_store.properties.UserRolesCacheEnabled": true,
+      "user_store.properties.ConnectionRetryDelay": 120000
```

Comment:
Add the unit here. 
`"user_store.properties.ConnectionRetryDelay": "2m"`

Reply:
Fixed